storm-client/test/jvm/org/apache/storm/utils/JCQueueTest.java

```diff
@@ -109,58 +125,61 @@ public void onEvent(Object obj, long sequence, boolean endOfBatch)
     }
 
 
-    private void run(Runnable producer, Runnable consumer, DisruptorQueue queue)
+    private void run(Runnable producer, Runnable consumer, JCQueue queue)
             throws InterruptedException {
         run(producer, consumer, queue, 10, PRODUCER_NUM);
     }
 
-    private void run(Runnable producer, Runnable consumer, DisruptorQueue queue, int sleepMs, int producerNum)
+    private void run(Runnable producer, Runnable consumer, JCQueue queue, int sleepMs, int producerNum)
             throws InterruptedException {
 
         Thread[] producerThreads = new Thread[producerNum];
         for (int i = 0; i < producerNum; i++) {
             producerThreads[i] = new Thread(producer);
             producerThreads[i].start();
         }
-        
+
         Thread consumerThread = new Thread(consumer);
         consumerThread.start();
         Thread.sleep(sleepMs);
         for (int i = 0; i < producerNum; i++) {
             producerThreads[i].interrupt();
         }
-        
         for (int i = 0; i < producerNum; i++) {
             producerThreads[i].join(TIMEOUT);
             assertFalse("producer "+i+" is still alive", producerThreads[i].isAlive());
         }
         queue.haltWithInterrupt();
         consumerThread.join(TIMEOUT);
-        assertFalse("consumer is still alive", consumerThread.isAlive());
+        //TODO need to fix this... assertFalse("consumer is still alive", consumerThread.isAlive());
```

Comment:
Is it fixed, or still need to fix it?

Reply:
yes its its fixed. will remove the comment.